modules/artifact-activation/src/main/java/org/mule/runtime/module/artifact/activation/api/extension/ExtensionModelLoaderRepository.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.runtime.module.artifact.activation.api.extension;
```

Comment:
move the 4 classes you added into this package to a subpackage called `boot` or `discovery` or `loader` or something like that

Reply:
Done.